core/jazz_services/components/validation/validate_create_payload.js

```diff
@@ -85,9 +85,8 @@ module.exports = (service_data, onComplete) => {
         },
         // Check if a service with same domain and service_name combination exists
         validateServiceExists: function (onComplete) {
-            getAllRecords = "true";
             var query = { service: service_data.service.toLowerCase(), domain: service_data.domain.toLowerCase() };
-            crud.getList(query, getAllRecords, function onServiceGet(error, data) {
+            crud.getList(query, null, function onServiceGet(error, data) {
```

Comment:
is this intended

Reply:
This is not intended. It is not being used after the acl update, so removed getAllRecords.
getList() expect the list of serviceIds as param which will be null for POST request.